pkg/api/mlflow/service/artifact/storage/storage.go

```diff
@@ -31,13 +32,15 @@ func (o ArtifactObject) IsDirectory() bool {
 	return o.IsDir
 }
 
-// ArtifactStorageProvider provides and interface to work with particular artifact storage.
+// ArtifactStorageProvider provides an interface to work with artifact storage.
 type ArtifactStorageProvider interface {
+	// Get will return a io.ReadCloser for specific artifact.
```

Comment:
can you please change `comment` to be like `Get returns a io.ReadCloser for specific artifact.`

Reply:
ok